provisioning/inventories/staging/host_vars/beta/vars

```diff
@@ -0,0 +1,29 @@
+---
+# System configuration
+mangaki_name: 'mangaki_staging'
+mangaki_source_install_type: 'package'
+mangaki_source_package: '{{ mangaki_source_home }}/{{ mangaki_source_package_local|basename }}'
```

Comment:
`mangaki_source_package_local` is not defined.
You may want to use rather `mangaki_source_package_src` there and define it also.

How are we going to create the tar file? Manually by `python setup.py sdist`?